scripts/extract_strings.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+# syntax:
+# extract_strings.sh [--mailer-repo ./fxa-auth-mailer] [--content-repo ./fxa-content-server] [--l10n-repo ./fxa-content-server-l10n] train_number
+
+function usage() {
+    echo "syntax:"
+    echo "extract_strings.sh [--mailer-repo ./fxa-auth-mailer] [--content-repo ./fxa-content-server] [--l10n-repo ./fxa-content-server-l10n] train_number"
+    exit 1
+}
+
+function check_folder() {
+    if [[ ! -d $1 ]]; then
+        echo "Error!"
```

Comment:
nit: add a more descriptive error here. `Error: failed to find a directory to extract from` or something